toolchain/lower/testdata/operators/assignment.carbon

```diff
@@ -13,70 +13,25 @@
 fn Main() {
   var a: i32 = 12;
   a = 9;
-  let _: i32 = a;
+  // CHECK:STDERR: assignment.carbon:[[@LINE+7]]:3: error: no return expression should be provided in this context [ReturnStatementDisallowExpr]
```

Comment:
These errors don't look intended. Added some suggestions.

Reply:
Yes, I missed this. Fixed.